boards/arm/stm32l496g_disco/doc/stm32l496g_disco.rst

```diff
@@ -0,0 +1,255 @@
+.. _stm32l496g_disco_board:
+
+ST STM32L496G Discovery
+########################
+
+Overview
+********
+
+The STM32L496G Discovery board features an ARM Cortex-M4 based STM32L496AG MCU
+with a wide range of connectivity support and configurations. Here are
+some highlights of the STM32L496G Discovery board:
+
+
+- STM32L496AGI6 microcontroller featuring 1 Mbyte of Flash memory and 320 Kbytes of RAM in an UFBGA169 package
+- 1.54 inch 240 x 240 pixel-TFT color LCD with parallel interface
+- SAI Audio CODEC, with a stereo headset jack, including analog microphone input
+- Stereo digital MEMS microphones
+- microSD™ card connector (card included)
```

Comment:
the microSD logo is trademarked, but the name "microSD" isn't.  (Even if it were, we generally just an * to indicate a third-party trademark.)   Drop the (tm) here.